test/static-code

```diff
@@ -72,7 +72,9 @@ if [ "${WITH_PARTIAL_TREE:-0}" = 0 ]; then
 	# https://github.com/microsoft/TypeScript/issues/30511
 	# We can't tell tsc to ignore the .d.ts in node_modules/ and check our
 	# own cockpit.d.ts, so we do two separate invocations as a workaround:
-	node_modules/.bin/tsc --typeRoots /dev/null --strict pkg/lib/cockpit.d.ts
+	if git ls-files | grep "^pkg/lib" >/dev/null; then
```

Comment:
hmm,  `[ -n "$(git ls-files pkg/lib)" ]` would be a bit cleaner, but good enough!

Reply:
Can adjust that, we didn't trigger `fedora-coreos/other` yet